Homework_5_6/Homework_5_6.md

```diff
@@ -0,0 +1,80 @@
+## Task 1
+
+### SCRUM CEREMONIES:
+
+- **Product backlog refinement** - it is the meeting before sprints where you take tasks from the backlog and you refine/specify details that are needed, between team and product owner.
+- **Sprint planning** - it is a meeting before sprint, where the goals of sprint and how goals are delivered are put in place.
+- **Daily scrum** - it is a meeting where the team gets together every day and every member says what they did yesterday, what they do that day and if there are any blockers.
+- **Sprint review** - meeting where the team comes together and sees the end-work/results and see if there are any changes needed.
+- **Sprint retrospective** - meeting where the team discusses what went great and what can be improved next time.
+
+### SCRUM ROLES:
+
+- **Scrum Master** - is a development team's team lead/supporter. Scrum Master initiates meetings, coaches, supports and protects the team, communicates, helps with blockers, reports. Scrum Master is between PM and the management.
+- **Product Owner** - responsible for the final product, takes feedback from different stakeholders and translates technical and business sides to each other.
+- **Development Team** - is a group of people who do the activities (planning, programming, testing, analyzing) to reach the needed goal.
+
+## Task 2
```

Comment:
7/7 great job for breaking down the task into stories, task and sub-tasks.

Reply:
Although, for the S2, you should add more API task there. Because when user create a booking, you need an API to store the information to the database.